nemo/collections/tts/torch/data.py

```diff
@@ -605,7 +605,7 @@ def general_collate_fn(self, batch):
         max_energies_len = max(energies_lengths).item() if Energy in self.sup_data_types_set else None
 
         if LogMel in self.sup_data_types_set:
-            log_mel_pad = torch.finfo(batch[0][2].dtype).tiny
+            log_mel_pad = torch.finfo(batch[0][4].dtype).tiny
```

Comment:
Why was this changed?

Reply:
log_mel_pad is the 4th tuple not the 2nd.